spec/support/api/schemas/autocomplete.json

```diff
@@ -32,7 +32,8 @@
         "properties": {
           "id": { "type": "integer" },
           "value": { "type": "string" },
-          "label": { "type": "string" }
+          "label": { "type": "string" },
+          "distance": {"type": "null"}
```

Comment:
this is now showing up in these results but doesn't cause issues in vets-website